dmoj/graders/standard.py

```diff
@@ -114,6 +114,7 @@ def _generate_binary(self):
         return executors[self.language].Executor(
             self.problem.id,
             self.source,
+            tempdir_tag=str(self.judge.submission.id),
```

Comment:
Should we also pass this to the signature grader?

Reply:
Maybe, but I don't like the design of this patch as-is anyway (need it to look into a JVM failure).